cv/qc-cv.obo

```diff
@@ -18,1967 +18,341 @@ import: https://raw.githubusercontent.com/ISA-tools/stato/dev/releases/1.4/stato
 
 [Term]
 id: MS:4000068
-name: Ambient humidity
-def: "The ambient relative humidity in percent (0,100) over one or more time points. The first column contains is the ambient humidity value(s). The second contains the respective retention time. A negative RT implies that the exact time of measurement is unknown." [PSI:QC]
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000026 ! environment metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000260 ! Percent
+name: number of PSM
+def: "The number of peptide-to-spectrum-matches the identification process provides (before FDR filtering)." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000069
-name: MS1 Total ion current chromatogram
-def: "The total ion current chromatogram of MS1 measurements. The first column contains the retention time values in seconds, the second column the corresponding relative intensities." [PSI:QC]
-comment: The metric enables to check visually e.g. MS1 TIC stability along the retention time and its absolute value compared to MS2 total ion chromatogram or other analyses MS1 TICs. TIC drops caused by e.g. spray stability issues or bubbles presence can be detected here.
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-is_a: MS:4000023 ! MS1 metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_relation MS:4000067 ! Total ion current chromatogram
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000109 ! Relative intensity
+name: number of identified spectra before FDR
+def: "Number of identified spectra the identification process provides." [PSI:QC]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_relation QC:4000012 ! single run based metric
+relationship: has_units UO:0000189 ! count unit
 
 [Term]
 id: MS:4000070
-name: MS2 Total ion current chromatogram
-def: "The total ion current chromatogram of MS2 measurements. The first column contains the retention time values in second, the second column the corresponding relative intensities." [PSI:QC]
-comment: The metric enables to check visually e.g. MS2 TIC absolute value compared to MS1 total ion chromatogram or other analyses MS2 TICs.
-is_a: MS:4000006 ! table
-is_a: MS:4000010 ! ID free
-is_a: MS:4000022 ! chromatogram metric
-is_a: MS:4000024 ! MS2 metric
-relationship: has_relation MS:4000013 ! single run based
-relationship: has_relation MS:4000067 ! Total ion current chromatogram
-relationship: has_column: MS:4000108 ! Retention time
-relationship: has_column: MS:4000109 ! Relative intensity
+name: number of identified spectra after FDR
```

Comment:
should we mention MS level here?
usually this means MS2, but would an MS1 spectrum + mass fingerprinting also count as 'identified' ?

Reply:
How do we deal with ion mobility? does every 'microscan' in an IM frame count? What if scans were accumulated to improve S/N to obtain an ID?